iceoryx_hoofs/functional/include/iox/detail/function_ref.inl

```diff
@@ -91,11 +91,13 @@ function_ref<ReturnType(ArgTypes...)>::operator=(function_ref<ReturnType(ArgType
 }
 
 template <class ReturnType, class... ArgTypes>
+// AXIVION Next Construct AutosarC++19_03-A15.4.2, AutosarC++19_03-A15.5.3, FaultDetection-NoexceptViolations : Intentional behavior. The library itself does not throw and on the implementation side a try-catch block can be used
```

Comment:
> The library itself does not throw and on the implementation side a try-catch block can be used

But when we use `noexcept` here a try-catch block cannot be used by the user. Or did you meant that the try-catch block would be inside the callable so that it does not throw an exception?

Reply:
Exactly :)